autoload/gh/issues.vim

```diff
@@ -194,17 +194,31 @@ function! gh#issues#new() abort
   call gh#gh#init_buffer()
   call gh#gh#set_message_buf('loading')
 
-  let m = matchlist(bufname(), 'gh://\(.\{-}\)/\(.\{-}\)/\(.*\)/issues/new$')
+  let m = matchlist(bufname(), 'gh://\(.\{-}\)/\(.\{-}\)/issues/new$')
+
+  let l:owner = m[1]
+  let l:repo = m[2]
```

Comment:
We doesn't use prefix `l:` in function local; Cause when defining variables doesn't use any prefix in function local,  then will be implicity be function local scope as use prefix `l:`

Reply:
thank, i didn't know about that ^^
fixed by this.